src/pages/login.svelte

```diff
@@ -416,11 +450,8 @@
             <ThemeButton />
             <MediaQuery query="(min-width: 536px)" let:matches>
                 {#if matches}
-                    <Button
-                        style="tertiary"
-                        size="regular"
-                        href="https://create.anchor.link/"
-                        target="_blank"><Icon name="plus" /><Text>New Account</Text></Button
+                    <Button style="tertiary" size="regular" on:action={createAccount}
```

Comment:
I think this button should be disabled and show some progress indication when the popup is open

Reply:
Disabling it makes sense, but what do you mean by progress indication?

Reply:
I think our buttons has a spinner or loading thing that can be enabled

Reply:
Like an animation?

Reply:
Hmm the problem with disabling the button is that I don't have a reliable way to reenable the button if a user closes the popup. Any ideas?

Reply:
I guess we could have a polling function in the account creation library that checks if the popup window is still opened 🤔

Reply:
Ok, I added that to the account creation library so the account creation buttons get reenabled when you close the popup window manually.